src/main/java/org/folio/service/inventory/InventoryHoldingManager.java

```diff
@@ -317,4 +320,43 @@ public Future<String> handleHoldingsRecord(CompositePoLine compPOL, Location loc
     }
   }
 
+  public Future<List<JsonObject>> getHoldingsForAllTenants(CompositePoLine poLine, RequestContext requestContext) {
```

Comment:
it sounds like for all tenants in the system, lets rename to getHoldingsForAllLocationTenants